test/commands/dev.test.ts

```diff
@@ -12,7 +12,7 @@ import * as ComponentBuilder from '../../src/dependency-manager/spec/utils/compo
 import { MOCK_API_HOST } from '../utils/mocks';
 
 // set to true while working on tests for easier debugging; otherwise oclif/test eats the stdout/stderr
-const print = false;
+const print = true; // TODO: undo
```

Comment:
Is this todo still outstanding? similar for others in this file?

Reply:
Missed this, fixing